src/functoria_tool.ml

```diff
@@ -78,7 +78,14 @@ module Make (Config : Functoria_sigs.CONFIG) = struct
     in
     Arg.(value & opt (some string) None & doc)
 
-  let () =
+  let color =
+    let enum = ["auto", None; "always", Some `Ansi_tty; "never", Some `None] in
+    let color = Arg.enum enum in
+    let alts = Arg.doc_alts_enum enum in
+    let doc = Fmt.strf "Colorize the output. $(docv) must be %s." alts in
+    Arg.(value & opt color None & info ["color"] ~doc ~docv:"WHEN")
```

Comment:
Please also put the right documentation section (see the various options above).